contributing/developing.md

```diff
@@ -30,20 +12,29 @@ $ yarn build
 
 ## Development
 
-Link the plugin from the top-level project directory and then run your command:
+### Building the Library
+Clone the project and `cd` into it:
+
+```
+$ git clone git@github.com:forcedotcom/salesforcedx-apex.git
+$ cd salesforcedx-apex
+```
+
+Ensure that you have [Yarn](https://yarnpkg.com/) installed, then run:
 
 ```
-$ yarn plugin:link
-$ sfdx force:apex:log:list -u myOrg@example.com
+$ yarn install
+$ yarn build
 ```
+### Using the Library 
 
 Install the library locally by adding this information to your project's `package.json`:
 
 ```
 "@salesforce/apex-node": "file://path/to/salesforcedx-apex/packages/apex-node"
```

Comment:
This seems Windows specific. Should that be noted here?

Reply:
Or could be nice to show the Mac way of doing it

Reply:
I updated it to add an example path for MacOS also